BedrockCommandQueue.cpp

```diff
@@ -97,6 +97,14 @@ void BedrockCommandQueue::push(BedrockCommand&& item) {
     _queueCondition.notify_one();
 }
 
+void BedrockCommandQueue::requeue(BedrockCommand&& item) {
```

Comment:
This is a lot of repeated code, why not just have `push` take a flag to reschedule?